packages/npm-dependencies-validation/test/index.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { getDependencyIssues } from "../src/index";
+import { expect, assert } from "chai";
+import * as fsPromises from "fs/promises";
+import { SinonMock, createSandbox } from "sinon";
+
+describe("index unit test", () => {
```

Comment:
the describe/context/it should be written in BDD style
- https://mochajs.org/#bdd
- https://en.wikipedia.org/wiki/Behavior-driven_development#Specification_as_a_ubiquitous_language (different DSL verbs, but same concepts)

Reply:
This comment is for **all** the spec files

Reply:
What exactly wrong in tests ? 
Let's talk about it